.travis.yml

```diff
@@ -1,13 +1,24 @@
+sudo: required
+dist: trusty
+group: edge
+
 language: go
 go:
     - 1.6.x
     - 1.7.x
     - 1.8.x
     - tip
+
 install:
     - go get -v golang.org/x/tools/cmd/cover
+    - sudo add-apt-repository -y ppa:opencpu/opencpu-1.6
```

Comment:
Oh. You've removed the stop server function as well. Is there another undocumented way to stop the server that is not a signal (sending signals makes session control way more complicated)?